specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/virtualWan.json

```diff
@@ -3197,6 +3435,17 @@
         "securityProviderName": {
           "type": "string",
           "description": "The Security Provider name."
+        },
+        "virtualHubRouteTableV2s": {
```

Comment:
Why are we exposing this as '...V2s'? v1 or v2 should be decided by api versions, not properties of the resource. Please call this 'virtualHubRouteTables'

Reply:
There is already a VirtualHubRouteTable defined as a property, which will be deprecated in the future. This is why we have decided to go ahead with this.

Reply:
We discussed and this will be changed in a future API version.